README.md

```diff
@@ -18,668 +17,216 @@ replacement for common `pip` and `pip-tools` workflows.
   </picture>
 </p>
 
-<p align="center">
-  <i>Installing the Trio dependencies with a warm cache.</i>
-</p>
-
 ## Highlights
 
-- ⚖️ Drop-in replacement for common `pip`, `pip-tools`, and `virtualenv` commands.
-- ⚡️ [10-100x faster](https://github.com/astral-sh/uv/blob/main/BENCHMARKS.md) than `pip` and
-  `pip-tools` (`pip-compile` and `pip-sync`).
-- 💾 Disk-space efficient, with a global cache for dependency deduplication.
-- 🐍 Installable via `curl`, `pip`, `pipx`, etc. uv is a static binary that can be installed without
-  Rust or Python.
-- 🧪 Tested at-scale against the top 10,000 PyPI packages.
-- 🖥️ Support for macOS, Linux, and Windows.
-- 🧰 Advanced features such as [dependency version overrides](#dependency-overrides) and
-  [alternative resolution strategies](#resolution-strategy).
-- ⁉️ Best-in-class error messages with a conflict-tracking resolver.
-- 🤝 Support for a wide range of advanced `pip` features, including editable installs, Git
-  dependencies, direct URL dependencies, local dependencies, constraints, source distributions, HTML
-  and JSON indexes, and more.
+- 🚀 A single tool to replace `pip`, `pip-tools`, `pipx`, `poetry`, `pyenv`, `virtualenv`, and more.
+- ⚡️ [10-100x faster](https://github.com/astral-sh/uv/blob/main/BENCHMARKS.md) than `pip`.
+- 🐍 [Installs and manages](#python-management) Python versions.
+- 🛠️ [Runs and installs](#tool-management) Python applications.
+- ❇️ [Runs scripts](https://docs.astral.sh/uv/guides/scripts), with support for
```

Comment:
Should we say "single-file scripts"?

Reply:
Sure, that copy makes sense elsewhere.. I think.